src/core/Akka/Actor/Settings.cs

```diff
@@ -234,6 +235,12 @@ private static string GetProviderClass(string provider)
         /// </summary>
         /// <value>The logger start timeout.</value>
         public TimeSpan LoggerStartTimeout { get; private set; }
+        
+        /// <summary>
+        ///     Gets the logger start timeout.
+        /// </summary>
+        /// <value>The logger start timeout.</value>
+        public bool LoggerAsyncStart { get; private set; }
```

Comment:
Can you try running it with this setting enabled to true everywhere? Looks like the test suite is running without it

Reply:
Yes - want to check if test suite will work well. Because it seems to me that it is blocking even without async version.
And before introducing this setting, the code was using "async" implementation by default (so I added this setting to be able to disable new behavior) - and test suite kind of stopped.

Probably I need to wait longer. Just checking out CI output and it does not change for some time